### PR TITLE
Adds support for OPS

### PR DESCRIPTION
**Pull request by JhonathanLepidus**
Hi!

We're sending this contribution so this plugin can support functioning in OPS too.

Best regards, Jhon.

### CODE REVIEW

**Comment by ajnyga** [comment]
Sorry Jhon for taking so long! Please feel free to ping me in Mattermost/Slack if you do not hear from me in these cases. Github updates are hard to follow at times.

I am having a meeting about this plugin next week and will add you changes when I start working on it again!

**Comment by JhonathanLepidus** [comment]
Great! Should I send a PR to the plugin-gallery, in order to make this release broadly available to OPS users?

**Comment by ajnyga** [comment]
Go ahead thanks! I am doing work on 3.4 right now.

**Comment by ajnyga** [comment]
oh wait, there might be a problem with using `get_template_vars` wonder if that is with 3.3 as well or just with 3.4

edit: yes it is deprecated but still working in 3.3, it is replaced with getTemplateVars in 3.4
